content/agent/proxy.md

```diff
@@ -0,0 +1,154 @@
+
+`agent ---> haproxy ---> Datadog`
+
+We assume that HAProxy is installed on a host that has connectivity to Datadog.  
+Use the following configuration file if you do not already have it configured.
+
+```
+# Basic configuration
+global
+    log 127.0.0.1 local0
+    maxconn 4096
+    stats socket /tmp/haproxy
+
+# Some sane defaults
+defaults
+    log     global
+    option  dontlognull
+    retries 3
+    option  redispatch
+    timeout client 5s
+    timeout server 5s
+    timeout connect 5s
+
+# This declares a view into HAProxy statistics, on port 3835
+# You do not need credentials to view this page and you can
+# turn it off once you are done with setup.
+listen stats :3835
+    mode http
+    stats enable
+    stats uri /
+
+# This declares the endpoint where your agents will connect.
+# In this example we use port 3834 but you can use any other
+# free port.
+frontend forwarder
+    bind *:3834 # DTDG
+    mode tcp
+    default_backend datadog
+
+# This is the Datadog server. In effect any TCP request coming
+# to the forwarder frontend defined above will be proxied to
+# Datadog's public endpoints.
+backend datadog
+    balance roundrobin
+    mode tcp
+    option tcplog
+    server mothership haproxy-app.agent.datadoghq.com:443 check port 80
+```
+
+Once the HAProxy configuration is in place, you can reload it or restart HAProxy.
+
+**We recommend having a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.com` fails over to another IP.
+
+Then edit each agent to point to HAProxy by setting its `dd_url` to the address of HAProxy (e.g. haproxy.example.com). This `dd_url` setting can be found in `datadog.conf`.
+
+`dd_url: https://haproxy.example.com:3834`
+
+Before you [restart the agent](/agent/faq/start-stop-restart-the-datadog-agent) Edit your supervisor configuration to disable SSL certificate verification. This is needed to prevent python from complaining about the discrepancy between the hostname on the SSL certificate (app.datadoghq.com) and your HAProxy hostname.
+
+####  On GNU/Linux, Mac OS X, FreeBSD, SmartOS:
+You need to edit the supervisor configuration found at:
+
+* `/etc/dd-agent/supervisor_ddagent.conf` on debian-based systems
+* `/etc/dd-agent/supervisor.conf` on redhat-based systems
+* `/opt/local/datadog/supervisord/supervisord.conf` on SmartOS
+* `/usr/local/etc/datadog/supervisord/supervisord.conf` on FreeBSD
+* `~/.datadog-agent/supervisord/supervisord.conf` on Mac OS X
+
+Assuming that file is found at `SUP_FILE`
+```bash
+sed -i 's/ddagent.py/ddagent.py --sslcheck=0/' SUP_FILE
+```
+
+#### On Windows (Starting from agent 3.9.2):
+
+Edit your configuration file (datadog.conf) and add this option:
+```
+skip_ssl_validation: yes
+```
+
+Finally [restart the agent](/agent/faq/start-stop-restart-the-datadog-agent).
```

Comment:
should the link be /agent/faq/start-stop-restart-the-datadog-agent/#windows

Reply:
👍